README_first.ipynb

```diff
@@ -1,60 +1,124 @@
 {
```

Comment:
<p>I don't think the emphasis should be on porting, what they want to do is use their model, porting is the unfortunate reality.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Do you have a suggestion for a different header title?</p><p> </p>